### ISSUE TITLE
Delay in message post

### ISSUE THREAD

**Issue by tomswinkels** [dmitryelj/RPi-P2000Receiver#9]
There is an delay of 12 seconds before sending a post.

Is this a bug?

**Comment by dmitryelj** [comment]
Not a bug because there can be several messages in a queue with the same body, but to different recipients. The delay can be adjusted btw.

**Comment by tomswinkels** [comment]
But 12 seconds for default are very high, is there not another way to map messages with the same body? Maybe look to PDW?

**Comment by dmitryelj** [comment]
True, maybe 3-4s will be enough. You can try to adjust "post_delay_s" parameter to any value you want.

**Comment by tomswinkels** [comment]
Yes. 

I don't no how exactly P2000 decode works, but the same messages to differences addresses (capcodes) are sending a message for each address?

Maybe the decoder get a frame or some other stuff to check it is a group message or it is the last message in a group?

**Comment by dmitryelj** [comment]
p2000.py is a wrapper around multimon-ng.

There was an issue posted, please check: https://github.com/EliasOenal/multimon-ng/issues/123

**Comment by tomswinkels** [comment]
Yeah i know. But i was thinking what is the best place that make the group message.

**Comment by dmitryelj** [comment]
If it will be implemented officially in multimon-ng, I can use it, and there will be no need for a delay in a python application more. For now, settings 3-4s delay in a script is the easiest workaround.
